pkg/asset/agent/manifests/agent.go

```diff
@@ -57,22 +72,22 @@ func (m *AgentManifests) Files() []*asset.File {
 
 // Load returns the manifests asset from disk.
 func (m *AgentManifests) Load(f asset.FileFetcher) (bool, error) {
-	yamlFileList, err := f.FetchByPattern(filepath.Join(clusterManifestDir, "*.yaml"))
-	if err != nil {
-		return false, errors.Wrap(err, "failed to load *.yaml files")
-	}
-	ymlFileList, err := f.FetchByPattern(filepath.Join(clusterManifestDir, "*.yml"))
-	if err != nil {
-		return false, errors.Wrap(err, "failed to load *.yml files")
-	}
-
-	fileList := append(yamlFileList, ymlFileList...)
-	if len(fileList) == 0 {
-		return false, nil
-	}
-
-	m.FileList = fileList
-	asset.SortFiles(m.FileList)
-
-	return true, nil
+	// yamlFileList, err := f.FetchByPattern(filepath.Join(clusterManifestDir, "*.yaml"))
+	// if err != nil {
+	// 	return false, errors.Wrap(err, "failed to load *.yaml files")
+	// }
+	// ymlFileList, err := f.FetchByPattern(filepath.Join(clusterManifestDir, "*.yml"))
+	// if err != nil {
+	// 	return false, errors.Wrap(err, "failed to load *.yml files")
+	// }
+
+	// fileList := append(yamlFileList, ymlFileList...)
+	// if len(fileList) == 0 {
+	// 	return false, nil
+	// }
+
+	// m.FileList = fileList
+	// asset.SortFiles(m.FileList)
+
+	return false, nil
```

Comment:
No, I also think that each individual asset should load/generate its own entiy, the AgentManifests in my view it's just the main collector of all the required inputs for the agent code. Eventually, the load here could be use for extra manifests, but since it's not needed right now I will remove the comments in the following PR